package/files/nginx-pubcloud/nginx-registry-https.conf

```diff
@@ -0,0 +1,138 @@
+upstream rmt {
+    server 127.0.0.1:4224;
+}
+
+upstream registry {
+    server 127.0.0.1:5000;
+}
+
+## Set a variable to help us decide if we need to add the
+## 'Docker-Distribution-Api-Version' header.
+## The registry always sets this header.
+## In the case of nginx performing auth, the header is unset
+## since nginx is auth-ing before proxying.
+map $upstream_http_docker_distribution_api_version $docker_distribution_api_version {
+  '' 'registry/2.0';
+}
+
+
+server {
+    listen 443 default_server  ssl;
+    listen [::]:443 default_server ssl;
+    server_name rmt;
```

Comment:
But rmt, i.e. access to the repositories is covered in the existing config file `rmt-server-pubcloud-https.conf` why do we have to repeat the settings here?